### ISSUE TITLE
How to insert Confluent magic bytes 0-4 for Schema Registry id?

### ISSUE THREAD

**Issue by DaveSlinn** [takebayashi/fluent-plugin-avro#11]
We are successfully formatting our message to Kafka using the fluentd-plugin-kafka and this formatter plugin, but we would like to have it send the avro record to Kafka with the 5 "magic" bytes as defined by Confluent for associated the avro messages with a schema in the schema registry.

https://docs.confluent.io/current/schema-registry/serializer-formatter.html#wire-format

Is this possible?

**Comment by takebayashi** [comment]
No, custom prefixes/suffixes such as Confluent's wire format are not currently supported in this plugin.

**Comment by avandych** [comment]
will it be supported?

**Comment by DaveSlinn** [comment]
My teammate forked this repo and added the ability to specify the schemaid in the fluent config here https://github.com/mstinglin/fluent-plugin-avro
